### DISCUSSION TITLE
error while running simulation

### DISCUSSION THREAD

**Discussion by kethan27** [pybamm-team/PyBaMM#3142] [Q&A]
I want to run simulation of this experiment
<img width="540" alt="image" src="https://github.com/pybamm-team/PyBaMM/assets/135948584/f31805ec-7608-4390-b759-d607f0a00d47">
but while running the simulation after 160 cycles I have encountered this error
<img width="551" alt="image" src="https://github.com/pybamm-team/PyBaMM/assets/135948584/d3cac0be-10b2-4ec3-a7db-aa573db78e35">
any idea how to resolve this error?. And this simulation is taking very long time how to speed up it.

thank you.

**Comment by brosaplanella** [comment]
From your screenshot I can't tell if it is an error or not: this is just a warning, but the simulation might not crash as the solver does multiple attempts before failing.

For ideas on how to speed up the solver, see [this notebook](https://docs.pybamm.org/en/latest/source/examples/notebooks/solvers/speed-up-solver.html). Note that speeding the solver is extremely dependent on the model, parameters, experiment... so there is not a single solution that fixes all the cases.
